web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -168,15 +210,33 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
       return;
     }
 
-    await removeReviewPeriod(
-      toDelete,
-      csrf
-    );
-    dispatch({ type: UPDATE_REVIEW_PERIODS, payload: periods.filter((period) => period?.id !== toDelete) });
+    await removeReviewPeriod(toDelete, csrf);
+    dispatch({
+      type: UPDATE_REVIEW_PERIODS,
+      payload: periods.filter(period => period?.id !== toDelete)
+    });
     handleConfirmClose();
   }, [csrf, periods, dispatch, toDelete, handleConfirmClose]);
 
+  const loadFeedbackTemplates = useCallback(async () => {
+    const res = await getAllFeedbackTemplates(csrf);
+    const templates = res.payload.data;
+    templates.sort((t1, t2) => t1.title.localeCompare(t2.title));
+    setTemplates(templates);
+  }, [csrf, dispatch]);
+
+  useEffect(() => {
```

Comment:
This `useEffect` is new. It determines whether the Save button should be disabled based on the values entered and selected by the user.